.github/workflows/docker_update_readme.yml

```diff
@@ -0,0 +1,18 @@
+name: Update Docker Hub Description
+on:
+  push:
+    branches:
+    - master
+
+jobs:
+  dockerHubDescription:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: Docker Hub Description
+      uses: peter-evans/dockerhub-description@v2.0.0
```

Comment:
`peter-evans/dockerhub-description` says it's v2.1.0 now...

Reply:
huh? this has nothing to do with the raspberry build ... 

this action ONLY updates the dockerhub description - so runtime is ... irrelevant, really (it runs in  a few seconds).

Reply:
nevertheless - updating the action makes sense.